src/ui/field_formats_mixin.js

```diff
@@ -0,0 +1,5 @@
+import { FieldFormatsService } from '../core_plugins/kibana/common/field_formats/field_formats';
+
+export function fieldFormatsMixin(kbnServer, server) {
+  server.decorate('server', 'fieldFormats', new FieldFormatsService());
```

Comment:
I think we could improve the ergonomics here a bit:

1. How about making more instances of the `FieldFormatService` that are configured with a `getConfig()` function, rather than having a singleton that taked `getConfig` as an argument?
1. If this followed the pattern `UiSettings` uses, and also depended on `UiSettings` formally, it could look like this:

    ```js
    // for use in the context of a request, the default context
    server.decorate('request', 'getFieldFormatService', async function () {
      return await server.fieldFormatServiceFactory(this.getUiSettingsService())
    })

    // for use outside of the request context, for special cases
    server.decorate('server', 'fieldFromatServiceFactory', async function (uiSettingsService) {
      const configVals = await uiSettings...;
      const getConfig = (key) => configVals[key];
      return new FieldFormatService({ getConfig });
    });
    ```